deploy/prod/cla-assistant.yaml

```diff
@@ -0,0 +1,74 @@
+replicaCount: 1
+service:
+  enabled: true
+  name: cla-assistent
+  port: 8080
+  type: ClusterIP
+  containerPort: 8080
+  tls:
+    enabled: true
+    issuerName: cloudflare
+image:
+  repository: ${ECR_REGISTRY}/cla-assistent-prod
+  pullPolicy: Always
+  tag: prod
+serviceAccount:
+  create: false
+ingress:
+  enabled: true
+  host: cla.daifoundation.org
+  annotations:
+    external-dns.alpha.kubernetes.io/cloudflare-proxied: "true"
+podAnnotations:
+  reloader.stakater.com/auto: "true"
+resources:
+  limits:
```

Comment:
Huge limit for both cpu and mem?

Reply:
Yes, on ECS these were even higher, I adjusted according to real monitoring data